src/modules/course/schema/review.schema.ts

```diff
@@ -0,0 +1,33 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+import { ReviewType } from '../review.enum';
+
+export type ReviewDocument = Review & Document;
+
+@Schema({ timestamps: true })
+export class Review {
```

Comment:
Add reviewer Id to the schema so that reviewer could be uniquely identified.

Reply:
![image](https://user-images.githubusercontent.com/59202075/123541718-89e2ac00-d763-11eb-892e-fbe36d6a7a04.png)

mongo id is already present in the populated object

Reply:
ok